lib/fog/aws/requests/storage/head_object_url.rb

```diff
@@ -0,0 +1,40 @@
+module Fog
+  module Storage
+    class AWS
+      module HeadObjectUrl
+        def head_object_url(bucket_name, object_name, expires, options = {})
```

Comment:
Possibly, but it is a small enough amount of code I'm not too concerned about it for now anyway.